package.json

```diff
@@ -38,10 +38,7 @@
     "test-debug": "node --inspect-brk node_modules/.bin/jest --runInBand"
   },
   "license": "MPL-2.0",
-  "repository": {
```

Comment:
Please remove all of the changes to the package.json, these aren't needed here, and in fact may break things.